app/services/exports/lettings_log_export_service.rb

```diff
@@ -196,6 +196,10 @@ def apply_cds_transformation(lettings_log, export_mode)
         attribute_hash["age#{index}"] = -9 if attribute_hash["age#{index}_known"] == 1
       end
 
+      attribute_hash["log_id"] = lettings_log.id
+      attribute_hash["created_by"] = lettings_log.created_by&.email
```

Comment:
can we return name/ID if email is nil?

Reply:
I don't think we allow user records without email, so should be fine
`t.string "email", default: "", null: false`